CMakeLists.txt

```diff
@@ -2,6 +2,17 @@ cmake_minimum_required(VERSION 3.13.4)
 
 project(SurfaceToolbox)
 
+#-----------------------------------------------------------------------------
```

Comment:
Since the project will be included as "remote module" in Slicer, I suggest we omit this section. One less things to maintain